### PR TITLE
Revert "Revert "Irminger 10 deploy csvs""

### PR DESCRIPTION
**Pull request by reedan88**
So a bit of an explanation: in a fit of merge madness I had merged the Irminger 10 dummy deploy csvs into main along with the Irminger calibrations and ASMAT3 and AST3a cross-mio updates. We didn't want the deploy csvs merged into oceanobservatories yet. So I "reverted" that merge. Now I'm "reverting the revert" to create a PR to merge the Irminger 10 deploy csv to stage for updates when we get the actual times/dates.

Sorry about this. It will need to be checked again BUT this the initial revert had already been approved, so it should be a quick double-check that they match.

### CODE REVIEW

File: deployment/GI02HYPM_Deploy.csv
Comment (cooleyky) [inline deployment/GI02HYPM_Deploy.csv:1]: looks good. no changes from original version found with fc

File: deployment/GI03FLMA_Deploy.csv
Comment (cooleyky) [inline deployment/GI03FLMA_Deploy.csv:1]: looks good. no changes from original version found with fc

File: deployment/GI03FLMB_Deploy.csv
Comment (cooleyky) [inline deployment/GI03FLMB_Deploy.csv:1]: looks good. no changes from original version found with fc

**Review by cooleyky — APPROVED** [review]
Looks good. I downloaded the original versions from #1214 and this version and compared locally with file compare (fc) in Windows command line.

**Comment by reedan88** [comment]
Thanks everyone! And this time I'm NOT going to get ahead of myself and merge it.

**Comment by rtravis-whoi** [comment]
Updates ready for review...this is a bit messy because of the 'reverting' stuff that happened w/ the pre-cruise updates, so everything is lumped in together - pre-cruise & post-cruise plus the other stuff I did on the original PR (MET sensor additions, RDB revision letters etc).
The ONLY items needing review this time are Irm 9 recovered mooring dates/times, and Irm 10 deployed mooring dates/times, lat/lon, depths.  Per Dee - no sensor swaps or SN changes.
Glider deploy csvs will be coming on a separate PR.
[QLR Mooring Tables_9-11.docx](https://github.com/WHOIGit/ooicgsn-asset-management/files/12578118/QLR.Mooring.Tables_9-11.docx)

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -388,54 +493,139 @@ AR60-01,,AR69-01,,GI01SUMO-SBD12-03-HYDGN0000,8,1,2021-08-12T17:04:00,2022-07-06
...
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA001,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METCTD-12755,59.944783,-39.5736,,1,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA002,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METBPR-00250,59.944783,-39.5736,,-5,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA003,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METHRH-00296,59.944783,-39.5736,,-5,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA004,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METLWR-00244,59.944783,-39.5736,,-5,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METPRC-00264,59.944783,-39.5736,,-5,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA006,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METSWR-00253,59.944783,-39.5736,,-5,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA007,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METWND-00258,59.944783,-39.5736,,-6,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-08-FDCHPA000,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-FDCHPB-03904,59.944783,-39.5736,,-6,2663,,,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv:631]: Looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 2]
```diff
@@ -388,54 +493,139 @@ AR60-01,,AR69-01,,GI01SUMO-SBD12-03-HYDGN0000,8,1,2021-08-12T17:04:00,2022-07-06
...
+AR69-01,,AR76-03,,GI01SUMO-SBD11-06-METBKA003,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-METHRH-00286,59.936183,-39.470155,,-5,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD11-06-METBKA004,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-METLWR-00247,59.936183,-39.470155,,-5,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD11-06-METBKA005,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-METPRC-00238,59.936183,-39.470155,,-5,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD11-06-METBKA006,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-METSWR-00293,59.936183,-39.470155,,-5,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD11-06-METBKA007,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-METWND-00240,59.936183,-39.470155,,-6,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD11-08-NUTNRB000,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGINS-NUTNRB-01088,59.936183,-39.470155,,1,2694,,,
+AR69-01,,AR76-03,,GI01SUMO-SBD12-00-DCLENG000,9,1,2022-06-30T15:54:00,2023-09-10T10:13:00,CGMGI-01SUMO-00009,,CGCON-BDCL12-50010,59.936183,-39.470155,,0,2694,,,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv:553]: Looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 3]
```diff
@@ -1,78 +1,87 @@
...
+AR76-03,,,,GI02HYPM-WFP02-00-WFPENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGCON-GWFPGX-45201,59.973718,-39.525233,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-01-FLORDL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-FLORDL-04013,59.973718,-39.525233,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-03-DOSTAL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-DOSTAL-02198,59.973718,-39.525233,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-CTDPFL-00150,59.973718,-39.525233,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-05-VEL3DL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-VEL3DL-01256,59.973718,-39.525233,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55092,59.973718,-39.525233,,150,2661,upward-looking,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55093,59.973718,-39.525233,,150,2661,downward-looking,,D
```
Comment (reedan88) [inline deployment/GI02HYPM_Deploy.csv:87]: Looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 4]
```diff
@@ -1,78 +1,87 @@
...
+AR69-01,,AR76-03,,GI02HYPM-WFP02-01-FLORDL000,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.969927,-39.489243,,2560,2676,,,D
+AR69-01,,AR76-03,,GI02HYPM-WFP02-03-DOSTAL000,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.969927,-39.489243,,2560,2676,,,D
+AR69-01,,AR76-03,,GI02HYPM-WFP02-04-CTDPFL000,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.969927,-39.489243,,2560,2676,,,D
+AR69-01,,AR76-03,,GI02HYPM-WFP02-05-VEL3DL000,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.969927,-39.489243,,2560,2676,,,D
+AR69-01,,AR76-03,,GI02HYPM-MPM01-02-ZPLSGA009,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,,CGINS-ZPLSGA-55109,59.969927,-39.489243,,150,2676,upward-looking,,D
+AR69-01,,AR76-03,,GI02HYPM-MPM01-02-ZPLSGA010,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,,CGINS-ZPLSGA-55111,59.969927,-39.489243,,150,2676,downward-looking,,D
```
Comment (reedan88) [inline deployment/GI02HYPM_Deploy.csv:78]: Looks good.

File: deployment/GI03FLMA_Deploy.csv [hunk 5]
```diff
@@ -1,235 +1,261 @@
...
+AR69-01,,AR76-03,,GI03FLMA-RIS01-00-SIOENG000,9,1,2022-07-03T13:45:00,2023-09-09T10:01:00,CGMGI-03FLMA-00009,,CGCON-SIOCON-14884,59.76685,-39.927187,,30,2690,deployed 10 m shallower than design depth of 2700 m due to error in waypoints table,,C
+AR69-01,,AR76-03,,GI03FLMA-RIS01-03-DOSTAD000,9,1,2022-07-03T13:45:00,2023-09-09T10:01:00,CGMGI-03FLMA-00009,,CGINS-DOSTAD-00457,59.76685,-39.927187,,30,2690,deployed 10 m shallower than design depth of 2700 m due to error in waypoints table,,C
+AR69-01,,AR76-03,,GI03FLMA-RIS01-04-PHSENF000,9,1,2022-07-03T13:45:00,2023-09-09T10:01:00,CGMGI-03FLMA-00009,,CGINS-PHSENF-P0181,59.76685,-39.927187,,30,2690,deployed 10 m shallower than design depth of 2700 m due to error in waypoints table,,C
+AR69-01,,AR76-03,,GI03FLMA-RIS01-05-FLORTD000,9,1,2022-07-03T13:45:00,2023-09-09T10:01:00,CGMGI-03FLMA-00009,,CGINS-FLORTD-01319,59.76685,-39.927187,,30,2690,deployed 10 m shallower than design depth of 2700 m due to error in waypoints table,,C
```
Comment (reedan88) [inline deployment/GI03FLMA_Deploy.csv:235]: Looks good.

File: deployment/GI03FLMA_Deploy.csv [hunk 6]
```diff
@@ -1,235 +1,261 @@
...
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG048,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13590,59.7711,-39.883083,,500,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH049,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13633,59.7711,-39.883083,,750,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH050,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-12219,59.7711,-39.883083,,1000,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH051,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13447,59.7711,-39.883083,,1500,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-00-SIOENG000,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGCON-SIOCON-13283,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-03-DOSTAD000,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-DOSTAD-00224,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-04-PHSENF000,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-PHSENF-P0141,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-05-FLORTD000,10,1,2023-09-05T14:57:00,,CGMGI-03FLMA-00010,,CGINS-FLORTD-01299,59.7711,-39.883083,,30,2691,,,C
```
Comment (reedan88) [inline deployment/GI03FLMA_Deploy.csv:261]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 7]
```diff
@@ -1,235 +1,261 @@
...
+AR69-01,,AR76-03,,GI03FLMB-RIM01-02-CTDMOH069,9,1,2022-07-02T13:43:00,2023-09-09T18:25:00,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12392,59.71513,-39.317078,,750,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIM01-02-CTDMOH070,9,1,2022-07-02T13:43:00,2023-09-09T18:25:00,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12390,59.71513,-39.317078,,1000,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIM01-02-CTDMOH071,9,1,2022-07-02T13:43:00,2023-09-09T18:25:00,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12222,59.71513,-39.317078,,1500,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIS01-00-SIOENG000,9,1,2022-07-02T13:43:00,2023-09-09T16:52:00,CGMGI-03FLMB-00009,,CGCON-SIOCON-12365,59.71513,-39.317078,,30,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIS01-03-DOSTAD000,9,1,2022-07-02T13:43:00,2023-09-09T16:52:00,CGMGI-03FLMB-00009,,CGINS-DOSTAD-00507,59.71513,-39.317078,,30,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIS01-04-PHSENF000,9,1,2022-07-02T13:43:00,2023-09-09T16:52:00,CGMGI-03FLMB-00009,,CGINS-PHSENF-P0182,59.71513,-39.317078,,30,2828,,,C
+AR69-01,,AR76-03,,GI03FLMB-RIS01-05-FLORTD000,9,1,2022-07-02T13:43:00,2023-09-09T16:52:00,CGMGI-03FLMB-00009,,CGINS-FLORTD-01435,59.71513,-39.317078,,30,2828,,,C
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:235]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 8]
```diff
@@ -1,235 +1,261 @@
...
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOG068,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOG-13623,59.719785,-39.3538,,500,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH069,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13631,59.719785,-39.3538,,750,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH070,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13454,59.719785,-39.3538,,1000,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH071,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13457,59.719785,-39.3538,,1500,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-00-SIOENG000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGCON-SIOCON-13989,59.719785,-39.3538,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-03-DOSTAD000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-DOSTAD-00226,59.719785,-39.3538,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-04-PHSENF000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-PHSENF-P0142,59.719785,-39.3538,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-05-FLORTD000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-FLORTD-01322,59.719785,-39.3538,,30,2820,,,C
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:261]: Looks good.

**Review by reedan88 — COMMENTED** [review]
Looks good. No errors found on deploys or recoveries.

File: deployment/GI01SUMO_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:631]: Deploy time and location looks good, recovery time looks good

File: deployment/GI02HYPM_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/GI02HYPM_Deploy.csv:87]: Deploy time and location looks good, recovery time looks good

File: deployment/GI03FLMA_Deploy.csv (hunk 5 above)
Comment (cdobs) [inline deployment/GI03FLMA_Deploy.csv:235]: Deploy time and location looks good, recovery time looks good

File: deployment/GI03FLMB_Deploy.csv (hunk 8 above)
Comment (cdobs) [inline deployment/GI03FLMB_Deploy.csv:261]: Deploy time and location looks good, recovery time looks good

**Review by cdobs — APPROVED** [review]
Reviewed deploy times and locations and recovery times. All look good

**Comment by rtravis-whoi** [comment]
[Irminger 10 AM Updates](https://trello.com/c/n3g5jRk2/198-irminger-10-am-updates)

**Pull request merged**